spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/iam/IdentityService.java

```diff
@@ -34,9 +39,19 @@
     /**
      * Verifies a JWT bearer token.
      *
-     *  @param token    the token to verify
+     * @param token The token to verify.
+     * @param additional Optional additional information required by the validations independent of the token.
      *
      */
-    Result<ClaimToken> verifyJwtToken(String token);
+    Result<ClaimToken> verifyJwtToken(String token, @Nullable Map<String, Object> additional);
```

Comment:
At this point I'd create a dedicated object that wraps the `token` and the `additional` map, it could extend or even re-use the existing `TokenRepresentation`

Reply:
Done - extended and used the TokenRepresentation